### PR TITLE
Update mcr.cpp

### PR DESCRIPTION
**Pull request by xunchelai**
update two

### CODE REVIEW

**Review by MuBai-Argo — COMMENTED** [review]
pass

**Comment by luxuehan913** [comment]
ok

**Review by whetheryoulikeornot — COMMENTED** [review]
1

**Review by luxuehan913 — COMMENTED** [review]
ok

**Review by MuBai-Argo — COMMENTED** [review]
pass

File: mcr.cpp [hunk 1]
```diff
@@ -12,7 +12,7 @@ bool isWin(char game[3][3]) {
         }
     }
     
-    // Check columns
```
Comment (GN-H) [inline mcr.cpp:15]: good

**Pull request closed without merging**